schemas/mw-page-edit.yaml

```diff
@@ -0,0 +1,70 @@
+title: MediaWiki Page Edit
+description: Represents a MW Page Edit event
+type: object
+properties:
+  # global event fields
+  meta:
+    type: object
+    properties:
+      uri:
+        type: string
+        format: uri
+        description: the unique URI identifying the event
+      request_id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique UUID v1 ID of the event derived from the X-Request-Id header
+      id:
+        type: string
+        pattern: '^[a-fA-F0-9]{8}(-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$'
+        description: the unique ID of this event; should match the dt field
+      dt:
+        type: string
+        format: date-time
+        description: the time stamp of the event, in ISO8601 format
+      domain:
+        type: string
+        description: the domain the event pertains to
+    required:
+      - uri
+      - request_id
+      - id
+      - dt
+      - domain
+  # event-specific fields
+  title:
+    type: string
+    description: the title of the edited page
+  page_id:
+    type: integer
+    minimum: 1
+    description: the page ID of the edited page
+  namespace:
+    type: integer
+    description: the namespace ID the page belongs to
+  revision:
+    type: integer
+    minimum: 1
+    description: the revision ID created by this edit event
+  base_revision:
+    type: integer
+    minimum: 1
+    description: the revision ID that the edit is based on
+  save_dt:
+    type: string
+    format: date-time
+    description: the time stamp of the revision in ISO8601 format
+  user_id:
+    type: string
```

Comment:
Should `user_id` be integer here?  MediaWiki has an integer ID, a user name (Eevans), and a full name (Eric Evans).